desktop/build.gradle

```diff
@@ -1,5 +1,5 @@
 plugins {
-    id 'edu.sc.seis.macAppBundle'
+//    id 'edu.sc.seis.macAppBundle'
```

Comment:
@pskowronek please fix macAppBundle. I've got error during gradle sync:
```
Caused by: groovy.lang.MissingPropertyException: Could not set unknown property 'destinationDir' for task ':pm-home-station:createAppZip' of type org.gradle.api.tasks.bundling.Zip.
	at org.gradle.internal.metaobject.AbstractDynamicObject.setMissingProperty(AbstractDynamicObject.java:118)
	at org.gradle.internal.metaobject.AbstractDynamicObject.setProperty(AbstractDynamicObject.java:76)
	at org.gradle.api.tasks.bundling.Zip_Decorated.setProperty(Unknown Source)
	at edu.sc.seis.macAppBundle.MacAppBundlePlugin$_createAppZipTask_closure70.doCall(MacAppBundlePlugin.groovy:256)
	````

Reply:
Pretty old https://plugins.gradle.org/plugin/edu.sc.seis.macAppBundle ...

Reply:
The maintainer doesn't support that anymore, but is kinda welcome for PRs. Yesterday tried to implement fixes into `macAppBundle` plugin - and I succeeded...... it generated DMG.... but .app is not working properly as `JavaAppLauncher` in it is probably deprecated under macOS. Will try to search for a quick solution, but I guess we may want to temporarily turn off .dmg generation (can you create an issue for it and add the link to commented out code).